README.md

```diff
@@ -537,7 +537,7 @@ Generates a schema object that matches an array data type. Note that undefined v
 Supports the same methods of the [`any()`](#any) type.
 
 ```javascript
-var array = Joi.array().includes(Joi.string().valid('a', 'b'));
+var array = Joi.array().items(Joi.string().valid('a', 'b'));
```

Comment:
Should become `Joi.array().items(Joi.string().valid('a', 'b').required());` not?